subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_verification.adoc

```diff
@@ -0,0 +1,849 @@
+- later on, the key is stolen and used to sign artifact `B`
+
+It means you can trust the key `A` for the first artifact, probably only up to the released version before the key was stolen, but not for `B`.
+====
+
+[[sec:ignoring-keys]]
+=== Specifying key servers and ignoring keys
+
+Gradle will automatically download the public keys required to verify a signature.
+For this it uses a list of well known and trusted key servers (the list may change between Gradle versions, please refer to the implementation to figure out what servers are used by default).
+
+You can explicitly set the list of key servers that you want to use by adding them to the configuration:
+
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<verification-metadata>
+   <configuration>
+      <verify-metadata>true</verify-metadata>
+      <verify-signatures>true</verify-signatures>
+      <key-servers>
+         <key-server uri="hkp://my-key-server.org"/>
+         <key-server uri="https://my-other-key-server.org"/>
+      </key-servers>
+   </configuration>
+</verification-metadata>
+----
+
+Despite this, it's possible that a key is not available:
+
+- because it wasn't published to a public key server
+- because it was lost
+
+In this case, you can ignore a key in the configuration block:
+
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<verification-metadata>
+   <configuration>
+      <verify-metadata>true</verify-metadata>
+      <verify-signatures>true</verify-signatures>
+      <ignored-keys>
+         <ignored-key id="abcdef1234567890" reason="Key is not available in any key server"/>
+      </ignored-keys>
+   </configuration>
+</verification-metadata>
+----
+
+As soon as a key is ignored, it will not be used for verification, even if the signature file mentions it.
+However, if the signature cannot be verified with at least one other key, Gradle will mandate that you provide a checksum.
+
+[[sec:local-keyring]]
+=== Exporting keys for faster verification
+
+Gradle automatically downloads the required keys but this operation can be quite slow and requires everyone to download the keys.
+To avoid this, Gradle offers the ability to use a local keyring file containing the required public keys.
+
+If the `gradle/verification-keyring.gpg` file is present, Gradle will search for keys there in priority.
+
+You can generate this file using GPG, for example issuing the following commands (syntax may depend on the tool you use):
+
+[source,bash]
+----
+$ gpg --no-default-keyring --keyring gradle/verification-keyring.gpg --recv-keys 379ce192d401ab61
+
+gpg: keybox 'gradle/verification-keyring.gpg' created
+gpg: key 379CE192D401AB61: public key "Bintray (by JFrog) <bintray@bintray.com>" imported
+gpg: Total number processed: 1
+gpg:               imported: 1
+
+$ gpg --no-default-keyring --keyring gradle/verification-keyring.gpg --recv-keys 6a0975f8b1127b83
+
+gpg: key 0729A0AFF8999A87: public key "Kotlin Release <kt-a@jetbrains.com>" imported
```

Comment:
Just in case, this is an official key as per https://youtrack.jetbrains.com/issue/KT-33781#focus=streamItem-27-3761490.0-0

Reply:
Is this a problem?

Reply:
No. It is not a problem. It might be a pointer for the reader that, well, this id indeed is used for the official Kotlin releases.